controllers/templatesync/template_sync.go

```diff
@@ -488,7 +589,7 @@ func (r *PolicyReconciler) Reconcile(ctx context.Context, request reconcile.Requ
 		}
 
 		// violation if object reference and policy don't match
-		if instance.GetName() != refName {
+		if !isClusterScoped && instance.GetName() != refName {
```

Comment:
We still need to catch the case for when the ConstraintTemplate was created outside of the policy or is a duplicate in general. I suggest this:
```suggestion
		if (parentPolicyLabel != "" && parentPolicyLabel != instance.GetName()) || (!isClusterScoped && instance.GetName() != refName) {
```

The code inside this if block would need to be adjusted slightly though.